packages/@okta/vuepress-site/docs/concepts/auth-overview/index.md

```diff
@@ -7,7 +7,7 @@ meta:
 
 # OAuth 2.0 Overview
 
-This page will give you an overview of OAuth 2.0 and OpenID Connect and their Okta implementations. It will explain the different flows, and help you decide which flow is best for you based on the type of application that you are building. If you already know what kind of flow you want, you can jump directly to:
+This page explains the differences between OAuth 2.0 and OpenID Connect and how these can be implemented using Okta. It will help you decide which flow is best for you based on the type of application that you are building. If you already know what kind of flow you want, you can jump directly to:
```

Comment:
I don't feel like this explains the difference BETWEEN OAuth 2.0 and OpenID Connect - more that it explains how they work with Okta. I like the original start to the sentence better:

This page provides an overview of OAuth 2.0 and OpenID Connect and how these can be implemented using Okta.......

Reply:
Text changed to : 
>This page provides an overview of OAuth 2.0 and OpenID Connect and how these different flows can be implemented using Okta. It will help you decide which flow is best for you based on the type of application that you are building.